documentation/specs/DisplayTable.md

```diff
@@ -0,0 +1,238 @@
+# `DisplayTable` Component Specification
+
+## Overview
+
+A `DisplayTable` is used for presenting information displayed across columns and rows.
+
+---
+
+## Design
+
+`DisplayTable` is for presentational concerns only. It doesn't support interactions such as selection, sorting, and row expansion. See the `Table` component for an interactive table accommodating advanced use cases.
+
+Without interaction, complexities are shifted to surfacing a simple component API, structuring the table for style requirements, and ensuring appropriate application of accessibility markup.
+
+`DisplayTable` will use React Aria's `useTable` under the hood to reliably generate accessibile markup for the table.
+
+`DisplayTable` will use `columns` and `rows` as the primary mechanism for passing data to the table. While this is a slight departure from our dot-notation pattern used throughout the rest of Easy UI, this makes for a consistent data input pattern with our `Table`-related components.
+
+### API
+
+```ts
+type DisplayTableProps = AriaLabelingProps & {
+  /** Columns for the table. */
+  columns: Column[];
```

Comment:
for `DisplayTable`, `columns` and `rows` follow a highly-specific data structure as noted in the type definitions below. this is a bit of a departure from our `Table` component, where the actual cell data is largely arbitrary and the rendering is controlled by the consumer (via `renderHeaderCell`, `renderBodyCell`, etc).

i think (a) i wanted to try to match our current MarketingTable API as much as we could where reasonable; and (b) it seems like to me for a display table where so much of the styling is governed by the internals of the component itself, it makes sense for it to not to defer rendering to the consumer and instead opt for a specific data structure.

let me know if that makes sense and feels cohesive enough in our component library for having the two approaches, and/or if there are better ideas